src/main/java/io/github/jasmin/controller/UserController.java

```diff
@@ -0,0 +1,39 @@
+package io.github.jasmin.controller;
+
+import io.github.jasmin.model.User;
+import io.github.jasmin.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+    @Autowired
+    private UserService userService;
+
+    @GetMapping("")
```

Comment:
```suggestion
    @GetMapping
```
제거해서 그대로 보여주는게 나은듯 하네요